packages/core/src/transport.ts

```diff
@@ -82,6 +84,20 @@ export class Batch<T> {
 
   private process(message: T) {
     const contextualizedMessage = lodashMerge({}, this.contextProvider(), message) as Context
+    if (
+      isObject(contextualizedMessage.view) &&
+      isObject(contextualizedMessage.view.measures) &&
+      isObject(contextualizedMessage.rum)
+    ) {
+      const loadEventEnd = contextualizedMessage.view.measures.load_event_end
+      if (typeof loadEventEnd === 'number' && loadEventEnd > 86400e9 /* 1 day in ns */) {
+        addMonitoringMessage(
+          `Buggy load event measure: ${loadEventEnd} - ` +
+            `View Id: ${contextualizedMessage.view.id} - ` +
+            `Document Version: ${contextualizedMessage.rum.document_version}`
```

Comment:
Do we not have all these elements already?

Reply:
This is right before serialization, so we should have it